ch03tests/05Mocks.ipynb

```diff
@@ -4,54 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>What does this try to show?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UCL/rsd-engineeringcourse/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
"**`unittest.mock` offers a base class for mocking objects called Mock. The following is an example how to define and object, call it and return id.**"